### DISCUSSION TITLE
Filling Missing Value, Capstone Project F&B TimeSeries.

### DISCUSSION THREAD

**Discussion by tu849035** [teamalgoritma/community#333] [Q&A] [answered]
Salam.

Saya melakukan pengisian missing value seperti tampilan di bawah.
![image](https://user-images.githubusercontent.com/107348211/191797623-012716eb-776d-4b81-8105-71b865f891a6.png)
Namun, setelah saya cek, masih terdapat missing value.
![image](https://user-images.githubusercontent.com/107348211/191798299-3511856a-ea63-4275-8b26-abc2582d2df2.png)

Saya isi missing value dengan nilai "0", karena data tersebut muncul akibat dari time series padding.
Yang memang seharusnya pada jam2 tersebut tidak ada pengunjung. Dan karena salah satu syarat time serises adalah tidak boleh ada periode waktu yg kosong. Namun, akibatnya saat pengisian missing value, masih terdapat nilai NA.

Mohon bantuan saran dan solusinya.

Terima Kasih.

**Comment by IrfanChairurrachman** [comment, marked as answer]
Dear @tu849035 ,

Kode screenshot pertama Anda telah benar dalam mengisi missing value, Anda juga telah mengisi missing value berdasarkan business wise yang benar yaitu mengisi dengan angka 0. Namun, pada kode pertama Anda belum menyimpan perubahan objek yang Anda lakukan dalam mengisi value. Sehingga Anda dapat menambahkan kode seperti ini:

`FnBtrain <- FnBtrain %>% mutate(visitor = na.fill(visitor, fill = 0))`

Kemudian silahkan cek apakah masih terdapat nilai Na atau tidak.

Sekian jawaban dari kami, jika merasa membantu silahkan tandai jawaban ini sebagai "Mark as Answer", namun jika masih terdapat kendala silahkan balas jawaban ini.
Terima kasih

**Reply by tu849035** [reply]
Astaga. Ternyata saya lupa itu. Terima Kasih atas bantuannya.
